packets/seedlings/dist/seedlings-marketing.css

```
@@ -7072,8 +7112,9 @@ Modifiers:
     --bottom: bottom only
     --left: left only
     --leaf: leaf frame
+    --hero_leaf: leaf frame top left only
     --x: rounded left & right
-Example: <div class="db square750 bg--neutral-200 overflow-hidden {{modifier}}"></div><p>{{label}}</p>
+    Example: <div class="db square750 bg--neutral-200 overflow-hidden {{modifier}}"></div><p>{{label}}</p> 
```

@monica-gr I'm not sure if indentation has any value in how these comments are parsed but you might want to outdent this example line to align with the other examples in other files.
```suggestion
Example: <div class="db square750 bg--neutral-200 overflow-hidden {{modifier}}"></div><p>{{label}}</p> 
```

---

@conwaydev does the indent do anything here? I called it out but only because I'm not sure how the example or modifier blocks are read by other services

---

Thank you!